torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1558,7 +1565,9 @@ static const auto DistributedC10dFrontendTorchBind =
                   c10::make_intrusive<::c10d::DistributedC10d>();
           return c10d_frontend_singleton;
         }))
-        .def("new_process_group_helper", &::c10d::DistributedC10d::newProcessGroupHelper);
+        .def(
```

Comment:
Nit: It would be great to have formatting-only changes in a separate diff if they are required.

Reply:
Sure. There are not too many such changes in this commit. Will try to avoid such case if too many such lines are involved.